app/models/stolen_record.rb

```diff
@@ -168,14 +185,26 @@ def tsv_row(with_article = true, with_stolen_locations: false)
     row
   end
 
+  def calculated_recovery_display_status
+    return "not_eligible" unless can_share_recovery
+    return "not_displayed" if not_displayed?
+    return "displayed" if displayed?
```

Comment:
Are you going to set this to `displayed` somewhere else in the future?

Reply:
No, the only place it gets set to displayed, currently, is when the display get's created.

Reply:
Great! That makes sense.